lib/pokerscore/texas_hold_em/hand_values.rb

```diff
@@ -0,0 +1,20 @@
+module TexasHoldEm
+  class HandValues
+    @@values = {
+      :high_card => 1,
+      :pair => 2,
+      :two_pair => 3,
+      :three_of_a_kind => 4,
+      :straight => 5,
+      :flush => 6,
+      :full_house => 7,
+      :four_of_a_kind => 8,
+      :straight_flush => 9,
+      :royal_flush => 10
+    }
+
+    def self.get_value(hand_type)
+      @@values.fetch(hand_type)
+    end
+  end
+end
```

Comment:
We could consider altering the public interface of this class.

Currently we use:
```
TexasHoldEm::HandValues.get_value(:high_card)
```

We could change the name up a little bit to be a little more concise, resulting in a nicer public interface.

The following doesn't repeat the term "value", and is a bit more specific as to what we're actually doing (getting the value of the hand _type_)
```
TexasHoldEm::HandTypeValue(:pair).call
```

(Yes, I'm advocating for another service object)

I'm also wondering if a constant may be favourable over a class variable here.